scripts.js

```diff
@@ -14,19 +14,20 @@
     });
 
   function processData(data) {
-    var drumsHTML = '';
+    let drumsHTML = '';
     keys = data;
 
```

Comment:
I tried to do `keys[...data]` here, but the object returned from the fetch promise is non-iterable.